docs/DafnyRef/Statements.md

```diff
@@ -2104,8 +2104,8 @@ an extended illustration of this technique to make all the dependencies of an `a
 
 ### 8.20.3. Revealing function bodies
 
-Normally function bodies are transparent and available for constructing proofs of assertions that use those functions.
-However, sometimes it is helpful to mark a function [`{:opaque}`](#sec-opaque) and treat it as an uninterpreted function, whose properties are
+By default, function bodies are transparent and available for constructing proofs of assertions that use those functions. 
+This can be changed using the `--defaul-function-opacity` commandline flag, or by using the [`:{opaque}`](#sec-opaque) attribute and treat it as an uninterpreted function, whose properties are
```

Comment:
```suggestion
This can be changed using the `--default-function-opacity` command line flag, or by using the [`:{opaque}`](#sec-opaque) attribute and treat it as an uninterpreted function, whose properties are
```

Also, should this reference the `opaque` keyword instead of the attribute now?

Reply:
Yes. I have moved the opaque attribute to the section of deprecated attributes, and changed the link to `#sec-opaque` to point to the section which discusses function transparency.